src/modules/did/did.processor.ts

```diff
@@ -42,28 +43,28 @@ export class DIDProcessor {
   }
 
   @OnQueueActive()
-  onActive(job: Job) {
-    this.logger.debug(`Starting ${job.name} document ${job.data}`);
+  onActive(job: Job<UpdateDocumentJobData>) {
+    this.logger.debug(`Starting ${job.name} document ${job.data.did}`);
   }
 
   @OnQueueStalled()
-  onStalled(job: Job) {
-    this.logger.debug(`Stalled ${job.name} document ${job.data}`);
+  onStalled(job: Job<UpdateDocumentJobData>) {
+    this.logger.debug(`Stalled ${job.name} document ${job.data.did}`);
   }
 
   @OnQueueFailed()
-  onFailed(job: Job) {
-    this.logger.debug(`Failed ${job.name} document ${job.data}`);
+  onFailed(job: Job<UpdateDocumentJobData>) {
+    this.logger.debug(`Failed ${job.name} document ${job.data.did}`);
   }
 
   @OnQueueWaiting()
-  async OnQueueWaiting(job: Job) {
-    this.logger.debug(`Waiting ${job.name} document ${job.data}`);
+  async OnQueueWaiting(jobId: number) {
```

Comment:
Argument to this hook differs from others https://docs.nestjs.com/techniques/queues#event-listeners-1